### PR TITLE
creates label component

### PR DESCRIPTION
**Pull request by gestchild**
References #3096

### CODE REVIEW

**Comment by gestchild** [comment]
There was a use case for a single label, which was why it was abstracted, but I think that is no longer the case. Is that right @davidpmccormick ?

File: common/views/components/Label/Label.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+  return (
+    <HtmlTag
+      href={label.url}
+      className={`
+      line-height-1
+      ${label.url ? 'plain-link font-white bg-green bg-hover-black' : 'font-black bg-yellow'}
+      ${font({s: 'HNM5'})}
+      ${spacing({s: 1}, {padding: ['top', 'bottom', 'left', 'right']})}
+    `} style={{display: 'block', whiteSpace: 'nowrap'}}>
```
Comment (davidpmccormick) [inline common/views/components/Label/Label.js]: Could use the `block` class which exists in root_scope_classes

**Comment by jamesgorrie** [comment]
> I think I feel happier with the idea that a label isn't always necessarily a list item

Why?

**Comment by davidpmccormick** [comment]
> Why?

Why would/should it be?

**Comment by jamesgorrie** [comment]
Why would having it available to use elsewhere make you feel happier.
My concern is then we use it out of context as to how it was designed to be used for i.e. in a list.

**Pull request merged**